test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -81,24 +83,32 @@ func (config *KubeletManagedHostConfig) verifyEtcHosts() {
 }
 
 func (config *KubeletManagedHostConfig) setup() {
-	etcHostContents := `127.0.0.1	localhost
+	var err error
+
+	// create a temporary /etc/hosts file if local /etc/hosts has the managed host header
+	hostsFileContent, err := ioutil.ReadFile("/etc/hosts")
+	if err != nil && bytes.HasPrefix(hostsFileContent, []byte(etcHostsPartialContent)) {
```

Comment:
if err!=nil, hostsFileContent would be invalid?

Reply:
oops!